src/components/create-vm-dialog/createVmDialog.jsx

```diff
@@ -925,7 +927,7 @@ class CreateVmModal extends React.Component {
                 storageSize: convertToUnit(this.state.storageSize, this.state.storageSizeUnit, units.GiB),
                 storagePool: this.state.storagePool,
                 storageVolume: this.state.storageVolume,
-                startVm: this.state.startVm,
+                startVm: this.state.unattendedInstallation || this.state.startVm,
```

Comment:
This is weird UI-wise -- it means you can uncheck the "immediately start" checkbox, but it will still auto-start the VM anyway, because it's unattended? If unattended install really doesn't work with a delayed start, then please add the reason as a comment here. But then the previous behaviour was actually right, and the checkbox *should* be auto-checked when you select unattended. But then it should also be disabled in that case.

Reply:
> This is weird UI-wise -- it means you can uncheck the "immediately start" checkbox, but it will still auto-start the VM anyway, because it's unattended? If unattended install really doesn't work with a delayed start, then please add the reason as a comment here. But then the previous behaviour was actually right, and the checkbox _should_ be auto-checked when you select unattended. But then it should also be disabled in that case.

No, that is exactly current behavior which this PR doesn't ammend, it was like that before this PR and will still be even with this PR. When unattended installation checkbox is checked, then the startVm checkbox is disabled and automatically shown as checked.

What I'm trying to change in this PR is that startVm value doesn't get overridden in two bugged cases:
1. uncheck "Start VM" checkbox, select OS with unattended installation, check "Unattended installation", uncheck "Unattended installation", "Start VM" checkbox is checked.
2. uncheck "Start VM" checkbox, select OS with unattended installation,  click "X" button, "Start VM" button is checked.
So to fix both cases, I changed
```
-  this.setState({ unattendedInstallation: value, startVm: true });
+ this.setState({ unattendedInstallation: value });
```
Because when you click on "X" button, it calls onValueChanged("os", null), which in turns call onValueChanged("unattendedInstallation", false), which calls the above mentioned code, which I think wasn't logical because what it basically did was that whenever state of "unattendedInstallation" changed, it just set startVm to true which was cases both above mentioned bugs.

Reply:
Makes sense, thanks! I tested this now, and it works correctly.